src/ui/view/modal/okp4-modal.scss

```diff
@@ -0,0 +1,37 @@
+@import '@/ui/style/theme.scss';
+@import '@/ui/style/mixins.scss';
+
+.okp4-dataverse-portal-okp4-modal-main {
+  @include with-theme {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    color: themed('text');
+    padding: 12px;
+    width: 358px;
```

Comment:
In mockups, modal's size is set to 358px/352px (according to screens' sizes), but here your width is bigger because of the padding.